common/src/main/java/com/viaversion/viaversion/ViaManagerImpl.java

```diff
@@ -153,12 +152,7 @@ public void onServerLoaded() {
         // Load Platform
         loader.load();
         // Common tasks
-        mappingLoadingTask = Via.getPlatform().runRepeatingSync(() -> {
-            if (protocolManager.checkForMappingCompletion()) {
-                mappingLoadingTask.cancel();
-                mappingLoadingTask = null;
-            }
-        }, 10L);
+        Via.getPlatform().runRepeatingSync(protocolManager::checkForMappingCompletion, 10L);
```

Comment:
This doesn't look right. The mappings cache use is a bit strange, admittedly, but I'll have to double check